content/docs/command-reference/remote/modify.md

```diff
@@ -406,7 +406,42 @@ application. To use a custom authentication method, use the following parameters
   ```dvc
   $ dvc remote modify myremote allow_anonymous_login true
   ```
+**Authentication examples:**
 
+Authentication example with `connection_string`:
+
+```dvc
```

Comment:
Maybe we should make an actual `## Example: Azure authentication methods` section at the bottom of this doc instead of expanding this `<details>` section?